packages/unified-pipeline/src/components/Canvas/Canvas.tsx

```diff
@@ -156,12 +154,7 @@ const CanvasInternal = (props: CanvasProps) => {
         onEdgesChange={onEdgesChange}
         onConnect={onConnect}
         proOptions={{ hideAttribution: true }}
-        fitView
-        onInit={instance =>
-          setTimeout(() => window.requestAnimationFrame(() => instance.fitView({ duration: ANIMATION_DURATION })), 0)
-        }
-        minZoom={0.5}
-        maxZoom={1}
+        minZoom={1}
```

Comment:
will this prevent the user from zooming out to see the whole pipeline?

Reply:
User can zoom-out only if zoomed-in already, i.e. if at zoom greater than 100%. We can remove this as well if needed.

Reply:
Removed it, this will allow zooming out beyond zoom level 100%.